app/Controllers/Http/ProductsController.ts

```diff
@@ -0,0 +1,17 @@
+import type { HttpContextContract } from '@ioc:Adonis/Core/HttpContext'
+import products from 'App/Resourses/ProductsDummyData'
+
+
+export default class ProductsController {
+  public async index(ctx: HttpContextContract) {
+    const query = ctx.request.qs()
+    const page: number = query.page || 1;
+    let sortedProducts = products.filter((item) => {
```

Comment:
There is not sorting handling.... the user can sort on the basis of price and  name, with both options asc and desc

use 2 query keys `sortBy` and `sortType` for the respective thingys

Reply:
kya hame is wale main ye sorting bhi lgani hai ??

Reply:
hn hn

Reply:
achha krta hu isko baki changes krke push krdiya hai